dashboards/operator.json

```diff
@@ -1,38 +1,4 @@
 {
-  "__inputs": [],
```

Comment:
It is weird these fields were deleted. What Grafana version was used for exporting this dash?

Reply:
Grafana v10.4.0